vulkan_generator/handle_remapper/generator.py

```diff
@@ -0,0 +1,299 @@
+# Copyright (C) 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This is the entry point for Vulkan Code Generator"""
+
+import os
+
+from pathlib import Path
+from typing import List
+
+from textwrap import dedent
+from vulkan_parser import type_parser
+from codegen_utils import codegen
+
+def is_dispatchable(handle: str) -> bool:
```

Comment:
Style:

As far as I can see in python, they do not use `is_` prefix.

Instead they write:

```
if dispatchable:
    # do stuff

if not dispatchable:
   # do stuff
```

Reply:
Function doesn't exist anymore